### ISSUE TITLE
notify_push bug in docker stack

### ISSUE THREAD

**Issue by raulrr1986** [nextcloud/notify_push#110]
| Backtrace omitted.
notify_push                   | Run with RUST_BACKTRACE=1 environment variable to display it.
notify_push                   | Run with RUST_BACKTRACE=full to include source snippets.
notify_push                   | Error: 
notify_push                   |    0: Failed to parse config
notify_push                   |    1: Failed to parse nextcloud config
notify_push                   |    2: Error while parsing php literal:  
notify_push                   |    2: 1 |  array (
notify_push                   |    2: 2 |     'mail_smtpmode' => 'smtp',
notify_push                   |    2: 3 |     'mail_smtphost' => getenv('SMTP_HOST'),                                                                                   
notify_push                   |    2:   |                        ^^ No valid token found, expected one of [Bool, Integer, Float, LiteralString, Null, Array, SquareOpen]
notify_push                   |    2: 4 |     'mail_smtpport' => getenv('SMTP_PORT') ?: (getenv('SMTP_SECURE') ? 465 : 25),
notify_push                   |    2: 5 |     'mail_smtpsecure' => getenv('SMTP_SECURE') ?: '',
notify_push                   |    2: 
notify_push                   | 
notify_push                   | Location:
notify_push                   |    src/config/nc.rs:8
notify_push                   | 
notify_push                   | Backtrace omitted.
notify_push                   | Run with RUST_BACKTRACE=1 environment variable to display it.
notify_push                   | Run with RUST_BACKTRACE=full to include source snippets.

**Comment by cryxy** [comment]
I have the same issues:

```
Backtrace omitted.,
Run with RUST_BACKTRACE=1 environment variable to display it.,
Run with RUST_BACKTRACE=full to include source snippets.,
Error: ,
   0: Failed to parse config,
   1: Failed to parse nextcloud config,
   2: Error while parsing php literal:  ,
   2: . |,
   2: 2 |   "apps_paths" => array (,
   2: 3 |       0 => array (,
   2: 4 |               "path"     => OC::$SERVERROOT."/apps",                                                                               ,
   2:   |                             ^^ No valid token found, expected one of [Bool, Integer, Float, LiteralString, Null, Array, SquareOpen],
   2: 5 |               "url"      => "/apps",,
   2: 6 |               "writable" => false,,
   2: ,
```

I think the issue is not docker related. It is related to #108

**Comment by aanno** [comment]
I encounter the _exactly_ same problem like @cryxy . The file causing the error is `config/apps.config.php`:

```bash
/var/www/html # grep SERVERROOT config/*.php
config/apps.config.php:              'path'     => OC::$SERVERROOT.'/apps',
config/apps.config.php:              'path'     => OC::$SERVERROOT.'/custom_apps',
```

Hence, it is like stated in #108: All `config/*.config.php` files are read from `notify_push` - even if the `--glob-config` flag is _**not**_ used.

**Comment by joshtrichards** [comment]
> Hence, it is like stated in #108: All config/*.config.php files are read from notify_push - even if the --glob-config flag is not used.

Fixed in #115
